keanu-project/src/test/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensorTest.java

```diff
@@ -46,35 +45,35 @@ public void setup() {
         matrixA = Nd4jDoubleTensor.create(new double[]{1, 2, 3, 4}, new long[]{2, 2});
         matrixB = Nd4jDoubleTensor.create(new double[]{1, 2, 3, 4}, new long[]{2, 2});
         scalarA = Nd4jDoubleTensor.scalar(2.0);
-        vectorA = Nd4jDoubleTensor.create(new double[]{1, 2, 3}, new long[]{3, 1});
-        vectorB = Nd4jDoubleTensor.create(new double[]{1, 2, 3}, new long[]{1, 3});
+        vectorA = Nd4jDoubleTensor.create(new double[]{1, 2, 3}, new long[]{3});
+        vectorB = Nd4jDoubleTensor.create(new double[]{1, 2, 3}, new long[]{3});
```

Comment:
`vectorA` and `vectorB` are only used in one test. They are now identical and you only need one of them.

Reply:
Got it. Good catch. Removed it.